### PR TITLE
link files in html and add css resets

### PR DESCRIPTION
**Pull request by HunterMuratore**
This PR links the files in html and adds css resets along with the theme color variables for the app

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+    
+
+
+    <script src="https://code.jquery.com/jquery-3.7.1.min.js" integrity="sha256-/JqT3SQfawRcv/BIHPThkBvs0OEvtFFmqPF/lYI/Cxo=" crossorigin="anonymous"></script>
+    <script src="js/script.js"></script>
+</body>
+</html>+
```
Comment (CoreyPhillipsOfficial) [inline index.html]: Extra + sign at end

**Review by CoreyPhillipsOfficial — COMMENTED** [review]
Almost!

File: index.html (hunk 1 above)
Comment (MEdelsteinE) [reply index.html]: On line 16 there's an extra + after the closing html tag

**Review by MEdelsteinE — CHANGES_REQUESTED** [review]
please remove that + on line 16

File: index.html (hunk 1 above)
Comment (HunterMuratore) [reply index.html]: fixed!

**Review by MEdelsteinE — APPROVED** [review]
Thank you!

**Pull request merged**